profiles/resources/default.cfg

```diff
@@ -1082,6 +1082,8 @@ SendPhishingAttackviaSET_____Phishing Email via SET_____@@@FRAMEWORK_DIR@@@/scri
 OpenSET_____Open the Social Engineering Toolkit (SET)_____cd @@@TOOL_SET_DIR@@@ ; ./set
 RCE_SSH_Connection_____Setup SSH Connection_____ssh -l @@@_RUSER@@@ @@@_RHOST@@@ -p @@@_RPORT@@@
 RCE_SBD_Connection_____Setup SBD Connection_____sbd -nv @@@_RHOST@@@ @@@_SBD_PORT@@@ -k @@@_SBD_PASSWORD@@@
+ExternalCommand_Injection_____Anastasios Stasinopoulos's Commix (Automated All-in-One OS Command Injection and Exploitation Tool)_____https://github.com/stasinopoulos/commix
+ExternalCommand_Injection_____Command_____python commix.py --url=@@@URL
```

Comment:
@stmh-infosec I don't think that ``@@@URL`` is a valid placeholder for the command. Maybe something like ``@@@target_url@@@`` maybe?